src/assets/icons/flags/de.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" id="flag-icon-css-de" viewBox="0 0 640 480">
```

Comment:
where do we get additional flags when adding locales? Does it really make sense to manage these files ourselves instead of through a library?

Reply:
The problem which I ran into with the library was that we either have to explicitly tell the service worker which flags to cache or it caches all flags which is around 30mb of storage. So I guess its not too much of a burden just download the library and copy the flag which is required when adding a new language.

Reply:
I think it is somewhat significant effort - and bears the risk that someone will add flags from a different source, making the design inconsistent.

how about instead setting the serviceworker to "lazy" for all flag resources (only cache once explicitly the file is requested)?

Similar could be an optimization for the double loading of es5 and es2018 script variants - but they do advice against it as es5 is anyway gonna be removed: https://github.com/angular/angular/issues/31256#issuecomment-506507021